Xamarin.Forms.Controls/CoreGallery.cs

```diff
@@ -485,8 +485,13 @@ public CoreRootPage(Page rootPage, NavigationBehavior navigationBehavior = Navig
 							GC.WaitForPendingFinalizers ();
 							GC.Collect ();
 						})
+					},
+					new Button {
+						Text = "Click to Open New Window",
+						Command = new Command(() => {
+							DependencyService.Get<IWindowNavigation>().OpenNewWindowAsync();
```

Comment:
`IWindowNavigation` service is only available in the UWP. On other platforms this will trows NRE.
```suggestion
							DependencyService.Get<IWindowNavigation>()?.OpenNewWindowAsync();
```

Reply:
Changed!